test/Test/Core/Issue.hs

```diff
@@ -0,0 +1,52 @@
+module Test.Core.Issue where
+
+import IW.Core.Id (Id (..))
+import IW.Core.Issue (Issue (..))
+import IW.Core.SqlArray (SqlArray (..))
+
+import Hedgehog
+import qualified Hedgehog.Gen as Gen
+import qualified Hedgehog.Range as Range
+
+testLabels :: [Text]
+testLabels = [ "good first issue"
+             , "help wanted"
+             , "low hanging fruit"
+             , "docs"
+             , "easy"
+             ]
+
+genIssue :: MonadGen m => m Issue
+genIssue = do
+    id' <- genId
+    number <- genNumber
+    title <- genTitle
+    body <- genBody
+    repoOwner <- genRepoOwner
+    repoName <- genRepoName
+    labels <- genLabels
+    -- | Construct issue url
+    let url = "https://github.com/" <> repoOwner <> "/" <> repoName <> "/issues/" <> show number
+
+    pure $ Issue id' number title body url repoOwner repoName labels 
```

Comment:
You can use the other side of `RecordWildCards` here. You can use it on creation if you name variables exactly like fields:

```haskell
issueId     <- genIssueId
issueNumber <- genIssueNumber
...
pure Issue{..}
```
Since all fields inside the `Issue` data type are strict, you will get an error if you forgot to initialise some fields :smirk:

Reply:
This is cool. Didn't know this.